### PR TITLE
[blog] data download tool post

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: _posts/2022-09-14-download-tool.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+author: Jennifer Chen
+title: New Data Download Tool
+---
+
+In the last year, we have added a ton of new data and many exciting new features. One such feature is the new Data Download tool that allows you to easily download statistical variable data for a large number of places with just a few button clicks.
+
+![](/assets/images/posts/download_tool.png)
```
Comment (juliawu) [inline _posts/2022-09-14-download-tool.md:9]: nit: add some alt-text inside the [] to improve accessibility

**Review by juliawu — APPROVED** [review]
Hooray for blog posts!

File: _posts/2022-09-14-download-tool.md (hunk 1 above)
Comment (chejennifer) [reply _posts/2022-09-14-download-tool.md]: done

File: _posts/2022-09-14-download-tool.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+author: Jennifer Chen
+title: New Data Download Tool
+---
+
+In the last year, we have added several interesting datasets and exciting new features to Data Commons. One such feature is the new Data Download tool that allows you to easily download statistical variable data for a large number of places with just a few button clicks.
```
Comment (beets) [inline _posts/2022-09-14-download-tool.md:7]: nit: we should link to the data download tool :)
```suggestion
In the last year, we have added several interesting datasets and exciting new features to Data Commons. One such feature is the new [Data Download tool](...) that allows you to easily download statistical variable data for a large number of places with just a few button clicks.
```

**Review by beets — APPROVED** [review]

File: _posts/2022-09-14-download-tool.md (hunk 2 above)
Comment (chejennifer) [reply _posts/2022-09-14-download-tool.md]: done

**Pull request merged**